pkg/reconciler/common/images_test.go

```diff
@@ -304,6 +311,13 @@ func validateUnstructedDaemonSetChanged(t *testing.T, tt *updateImageTest, u *un
 	util.AssertDeepEqual(t, daemonSet.Spec.Template.Spec.Containers, tt.expected)
 }
 
+func validateUnstructedJobChanged(t *testing.T, tt *updateImageTest, u *unstructured.Unstructured) {
```

Comment:
@aliok Would you mind doing a search for `unstructed` across all files. I think we should use `Unstructured`.

Reply:
Good catch :) I did a search&replace now in the entire codebase